src/redux/Missions/missions.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable camelcase */
+// import axios from 'axios';
```

Comment:
- [ ] Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable.